extra-admin/throttled/autobuild/prepare

```diff
@@ -0,0 +1,7 @@
+abinfo "Modifying systemd service file..."
+# Need update if upstream releases new package
+sed -i "s|ExecStart=.*|ExecStart=/bin/python3 /usr/lib/python$ABPY3VER/site-package/throttled.py|" "$SRCDIR"/systemd/lenovo_fix.service
```

Comment:
For `sed` actions, we need to write them like this, which may be more clear to see what happened:

```suggestion
sed -e "s|ExecStart=.*|ExecStart=/bin/python3 /usr/lib/python$ABPY3VER/site-package/throttled.py|" \
    -i "$SRCDIR"/systemd/lenovo_fix.service
```

And here wouldn't it be more appropriate to create a symlink at `/usr/bin` to site-package path?